assets/scss/bulmaswatch/_overrides.scss

```diff
@@ -0,0 +1,145 @@
+// Overrides
+@if $bulmaswatch-import-font {
+    @import url( 'https://fonts.googleapis.com/css?family=Muli:400,700');
+}
+
+.content blockquote {
+  border-color: $primary;
+}
+
+.button,
+.control.has-icons-left .icon,
+.control.has-icons-right .icon,
+.input,
+.pagination-ellipsis,
+.pagination-link,
+.pagination-next,
+.pagination-previous,
+.select,
+.select select,
+.textarea {
+  height: 2.572em;
+}
+
+.button {
+  &.is-active,
+  &:active {
+    box-shadow: inset 1px 1px 4px rgba($grey-darker, 0.3);
+  }
+  @each $name, $pair in $colors {
+    $color: nth($pair, 1);
+    $color-invert: nth($pair, 2);
+
+    &.is-#{$name} {
+      &.is-hovered,
+      &:hover {
+        background-color: darken($color, 10);
+      }
+
+      &.is-active,
+      &:active {
+        box-shadow: inset 1px 0 3px rgba($grey-darker, 0.3);
```

Comment:
Properties should be ordered background-color, box-shadow

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)